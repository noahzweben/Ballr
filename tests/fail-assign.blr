gameboard g1{
	clr = (251,142,74);
	size = (680,420);
	init -> {
		int x; 
		bool c; 

		c= 43; 
	}

}