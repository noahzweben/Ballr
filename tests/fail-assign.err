Fatal error: exception Failure("illegal assignment bool = int in c = 43")
make: *** [fail-assign.test] Error 2
