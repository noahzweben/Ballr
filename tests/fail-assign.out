Fatal error: exception Failure( "illegal assignment bool=int in c=43")